services/platform/platform.go

```diff
@@ -96,16 +96,17 @@ func (s *Service) Run(ctx context.Context) {
 	}
 }
 
-// SignUp creates new Percona Platform user with given email and password.
-func (s *Service) SignUp(ctx context.Context, email, password string) error {
+// SignUp creates new Percona Platform user with given email, firstName and lastName.
+func (s *Service) SignUp(ctx context.Context, email, firstName, lastName string) error {
 	cc, err := saasdial.Dial(ctx, "", s.host)
 	if err != nil {
 		return errors.Wrap(err, "failed establish connection with Percona")
 	}
 	defer cc.Close() //nolint:errcheck
 
-	_, err = api.NewAuthAPIClient(cc).SignUp(ctx, &api.SignUpRequest{Email: email, Password: password})
+	_, err = api.NewAuthAPIClient(cc).SignUp(ctx, &api.SignUpRequest{Email: email, FirstName: firstName, LastName: lastName})
 	if err != nil {
+		s.l.Error("pplatform", err)
```

Comment:
`pplatform`? can we add better error message?

Reply:
oh that was just added for debugging, forgot to remove it. will remove it

Reply:
removed